src/packages/@ncigdc/components/CartDownloadDropdown.js

```diff
@@ -49,16 +49,7 @@ const styles = {
 };
 
 const downloadCart = (user, files, dispatch, setState) => {
```

Comment:
can this be changed to keywords while you're here ?

Reply:
updated